iothub/service/src/Http2/HttpClientFactory.cs

```diff
@@ -0,0 +1,44 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+using System;
+using System.Net.Http;
+
+namespace Microsoft.Azure.Devices.Http2
+{
+    internal class HttpClientFactory
```

Comment:
Can we get meaningful doc comments for these classes, to help future SDK devs understand the class's role and purpose?